### ISSUE TITLE
buster-server says "running" even if address is already in use

### ISSUE THREAD

**Issue by dwittner** [busterjs/buster#432]
Starting `buster-server` twice produces the following output for me on my windows machine:

```
Address already in use. Pick another port with -p/--port to start buster-server
buster-server running on http://localhost:1111
```

The second message isn't displayed on my linux machine. I don't know whether it is the OS or node.js version which makes the different.

I have seen that we use `process.nextTick()` in [buster-server-cli\lib\server-cli.js](https://github.com/busterjs/buster-server-cli/blob/master/lib/server-cli.js#L49). I think using a callback for [server.listen](https://github.com/busterjs/buster-server-cli/blob/master/lib/server-cli.js#L71) instead, would be better and solve this problem.

**Comment by GCheung55** [comment]
Please take a look at the PR.

**Comment by dwittner** [comment]
Fixed in version 0.3.2 of **buster-server-cli**.
